README.md

```diff
@@ -31,7 +31,8 @@ Testing dependencies:
 1. Make sure Postgres and Redis are running. For example, on OS X:
 
   ```
-  $ brew services start redis postgres
+  $ brew services start redis 
+  $ brew services start postgres
```

Comment:
Good point. The previous sentence should probably say "For example, if you've installed the laptop script on OS X, you can start the services like this:"